Description: changes the modules we use
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/debian/etc/brokers/broker.cfg
+++ b/debian/etc/brokers/broker.cfg
@@ -37,7 +37,7 @@ define broker {
     # - glpidb                  = Save data in GLPI MySQL database
     # - Trending		= Save perfdata into a trending database
 #    modules     Livestatus, Simple-log, WebUI
-    modules WebUI, PickleRetentionBroker
+    modules PickleRetentionBroker, Livestatus, Graphite, SimpleLog, BoosterNrpe
 
     ## Advanced
     realm   All
--- a/debian/etc/pollers/poller.cfg
+++ b/debian/etc/pollers/poller.cfg
@@ -31,7 +31,7 @@ define poller {
     #                       should you desire it.
     # - SnmpBooster     = Snmp bulk polling module
     #modules     NrpeBooster, CommandFile
-    modules     
+    modules     NrpeBooster
 
     ## Advanced Features
     #passive         0       ; For DMZ monitoring, set to 1 so the connections
